ControlRoom/Simulator UI/ControlScreens/LocationView.swift

```diff
@@ -8,12 +8,15 @@
 
 import MapKit
 import SwiftUI
+import CoreLocation
 
 /// Map view to change simulated user's position
 struct LocationView: View {
     @ObservedObject var controller: SimulatorsController
     let simulator: Simulator
 
+    @State private var latitudeText = "37.323056"
+    @State private var longitudeText = "-122.031944"
     /// The location that is being simulated
     @State private var currentLocation = MKCoordinateRegion(
         center: CLLocationCoordinate2D(latitude: 37.323056, longitude: -122.031944),
```

Comment:
Might want to use constants here and in lines 18, 19 to easily change those in the future.

e.g.

```
DEFAULT_LAT = 37.323056
DEFAULT_LNG = -122.031944
```

18, 19 become:
```
    @State private var latitudeText = "\(DEFAULT_LAT)"
    @State private var longitudeText = "\(DEFAULT_LNG)"
```

```suggestion
        center: CLLocationCoordinate2D(latitude: DEFAULT_LAT, longitude: DEFAULT_LNG),
```

Reply:
We cannot use instance member 'DEFAULT_LAT' within property initializer; property initializers run before 'self' is available.